src/components/Movie/MovieDetails.js

```diff
@@ -34,16 +33,10 @@ export class MovieDetails extends Component {
 }
 
 function mapStateToProps(state, ownProps) {
-    return {
-        movie: selectors.getMovie(state),
-        selectedGenre: selectors.getSelectedGenre(state),
-        similarMovies: selectors.getSimilarMovies(state)
-    }
+    return { }

```

Comment:
In this case, you can just remove the whole function and call connect fn like this:

```javascript
export default connect(null, mapDispatchToProps)(MovieDetails);
```

Reply:
Okay.